source/layouts/topics.haml

```diff
@@ -13,8 +13,8 @@
 
   .container
     .row
-      .col-xs-3
-        = partial 'support_sidebar'
+      .col-xs-2.hidden-xs
+        / = partial 'support_sidebar'
```

Comment:
Thoughts on leaving the support sidebar in place (in addition to the footer) for topic index pages?

See e.g. https://support.aptible.com/topics/paas — since the footer falls well below the fold, I feel like the sidebar could still be helpful, and I'm OK with the redundancy.